src/point.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+from typing import Tuple
+
+PointType = Tuple[int, int]
```

Comment:
we don't use `PointType` anymore -- please remove

Reply:
Why? We will use it in the next PR

Reply:
fixed